src/TLuaInterpreter.cpp

```diff
@@ -14399,7 +14399,7 @@ int TLuaInterpreter::setServerEncoding(lua_State* L)
     }
 
     lua_pushnil(L);
-    lua_pushfstring(L, results.second.toLatin1().constData());
+    lua_pushfstring(L, results.second.toUtf8().constData());
```

Comment:
The names of the encodings are provided by Qt (IIRC) and they are confined to the ASCII or Latin1 encoding - so this would warrant being kept to the original form.